src/CollectdWinService/config/WindowsPerformanceCounter.config

```diff
@@ -1,47 +1,63 @@
 ﻿<WindowsPerformanceCounter>
-  <ReloadConfiguration Enable="true" Interval="900" />
+  <RefreshInstancesConfiguration Enable="true" Interval="900" />
   <Counters>

```

Comment:
you don't have the latest changes, for example "aggregation" was replaced but you still have it.